pkg/hostagent/netattachdef.go

```diff
@@ -175,11 +175,11 @@ func (agent *HostAgent) getPodResource(metadata *md.ContainerMetadata) error {
 		if agent.netattdefmap[netAttName] != nil {
 			isAcicniNetwork = true
 		} else {
-			return nil
+			return fmt.Errorf("Network Attachment Definition CR not applied: Must mention ACICNI plugin")
 		}
 	}
 	if isAcicniNetwork {
-		podResourceSock := filepath.Join(kubeletPodResourceDefaultPath, podresources.Socket+".sock")
+		podResourceSock := path.Join(kubeletPodResourceDefaultPath, podresources.Socket+".sock")
```

Comment:
shouldnt we be using filepath when working with files ?

Reply:
For Linux Operating systems, either of FilePath or Path package can be used. Paths are always separated by forward slashes, such as the paths in URLs in Linux. 
The Filepath package uses either forward slashes or backslashes, depending on the operating system. To process paths such as URLs that always use forward slashes regardless of the operating system, we use path package.
Usage of filepath or path doesnt make any difference for linux operation  :-)  I can revert it filepath anyway.

Reply:
I was just curious. either is fine to me.